extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -413,6 +425,14 @@ export class ConsentUI {
     this.isFullscreen_ = true;
   }
 
+  /**
+   * Send viewer a message.
+   * @param {string} message
+   */
+  sendViewerMessage_(message) {
```

Comment:
Nit: this signature is confusing, you're sending an `event` instead of a `message` (the message is most likely to be interpreted as the empty `dict()`)